ani.ter-markosyan/repo/Lesson_6/exercise_11/exercise_11.py

```diff
@@ -0,0 +1,10 @@
+#!usr/bin/evn python3
+
+list = []
+word = str
+while word != "":
+    word = input("Enter the words(enter a blank to exit): ")
+    if word not in list and word!="":
```

Comment:
LW: You should be able to check the blank line only in the while statement. Remove it from here.